### PR TITLE
Add Salzkotten

### PR DESCRIPTION
**Pull request by HerHde**
Add Salzkotten in the district of Paderborn, NRW, Germany as a stand-alone city (as I don't see an opportunity to add another `data_source` to Paderborn) with it's weekly market.

Ads on the rc3 made me aware of this project.

### CODE REVIEW

**Review by mojoaxel — COMMENTED** [review]
LGTM

**Review by mojoaxel — APPROVED** [review]

**Comment by johnjohndoe** [comment]
@HerHde Thank you for your contribution. Multiple data sources was discussed in issue #288 before. In the end nobody took action to implement it. If you want to work on this you can re-open the issue. In the meantime we can add Salzkotten as its own entity.

File: README.md [hunk 1]
```diff
@@ -314,6 +315,8 @@ state as the new version of the website available under [https://wo-ist-markt.de
 [rostock-markets]: http://www.rostocker-wochenmaerkte.de/standorte-angebote/
 [saarbruecken-wikipedia]: https://de.wikipedia.org/wiki/Saarbr%C3%BCcken
 [saarbruecken-markets]: https://www.saarbruecken.de/leben_in_saarbruecken/einkaufen/maerkte
+[salzkotten-wikipedia]: https://de.wikipedia.org/wiki/Salzkotten
+[salzkotten-markets]: http://vv.salzkotten.de/vv/produkte/174030100000004816.php
```
Comment (johnjohndoe) [inline README.md:319]: Please use `https://` here.

File: cities/salzkotten.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          8.6046,
+          51.6711
+        ]
+      }
+    }
+  ],
+    "metadata": {
+        "data_source": {
+            "title": "Stadt Salzkotten",
+            "url": "http://vv.salzkotten.de/vv/produkte/174030100000004816.php"
```
Comment (johnjohndoe) [inline cities/salzkotten.json:23]: Please use `https://` here.

**Comment by HerHde** [comment]
> If you want to work on this you can re-open the issue.

Thanks, providing some data is enough right now ;-)

**Review by johnjohndoe — APPROVED** [review]
LGTM. Thank you for the contribution.

Please note you can also add a date to the data source as suggested in the contribution guide to indicate to users how up-to-date the information is.

**Pull request merged**